app/models/experimentation/notifications_experiment.rb

```diff
@@ -21,15 +21,15 @@ def notifying?
     # See https://docs.google.com/document/d/1IMXu_ca9xKU8Xox_3v403ZdvNGQzczLWljy7LQ6RQ6A for more details.
     def self.conduct_daily(date)
       time(__method__) do
-        enrolling.each { |experiment| experiment.enroll_patients(date) }
+        enrolling.each { |experiment| experiment.enroll_patients(date, experiment.filters) }
         monitoring.each { |experiment| experiment.monitor }
         notifying.each { |experiment| experiment.schedule_notifications(date) }
       end
     end
 
     # Returns patients who are eligible for enrollment. These should be
     # filtered further by individual notification experiments based on their criteria.
-    def self.eligible_patients
+    def self.eligible_patients(filters = {})
```

Comment:
Unit tests are failing. Will remove older cases, and test newer cases before merging.

Reply:
Have added unit tests for new design.